src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -4128,6 +4169,14 @@ private void GetSlotsToMarkAsNotNullable(BoundExpression operand, ArrayBuilder<i
                             // access expressions.
                             break;
 
+                        case BoundKind.InterpolatedStringArgumentPlaceholder:
+                            if (TryGetPlaceholderSlot(operand, out slot) && PossiblyNullableType(operand.Type))
```

Comment:
>if (TryGetPlaceholderSlot(operand, out slot) && PossiblyNullableType(operand.Type)) [](http://example.com/codeflow?start=28&length=83)

It feels like we wouldn't need this special case if MakeSlot took care of the side lookup.